elements/designer-element-palette/designer-element-palette.html

```diff
@@ -64,10 +59,16 @@
     <h1>Elements</h1>
     <div id="elements">
       <template is="dom-repeat" items="{{builtIns}}">
-        <div class="element" on-mousedown="_onMouseDown">&lt;<span>{{item}}</span>&gt;</div>
+        <div class="element" on-mousedown="_onMouseDown">
```

Comment:
Yeah... given that I haven't used gestures yet, it might be better to go through everything at once. I have serious doubts about the usability of Designer via touch without much more serious though and design into making the whole UI touch-friendly - it's not going to be just handling events.